app/views/articles/index.html.erb

```diff
@@ -1,30 +1,12 @@
 <h1>Articles</h1>
 
-
-
-<h2>Our blog has <%= Article.count_without_status('archived') %> articles and counting!</h2>
-<h3>Blog current articles:</h3>
-<ul>
-  <% @articles.each do |article| %>
-    <% unless article.archived? %>
-      <li>
-        <%= link_to article.title, article %>
-      </li>
-    <% end %>
-  <% end %>
-</ul>
-
-<%= link_to "New Article", new_article_path %>
+<h2>Our blog has <%= Article.count_without_status('archived') %> active articles and counting!</h2>
```

Comment:
@JsMontesR Is it ok having logic in the views? What do you think?

Reply:
That's horrible!, I'd prefer to create a global helper (if it is possible) something like "count_archived_articles" and handle its logic from outside the view.

Reply:
@JsMontesR that's correct, please fix it in your code!

Reply:
✅